src/vivarium/framework/values.py

```diff
@@ -170,6 +172,29 @@ def union_post_processor(values: List[NumberLike], _) -> NumberLike:
     return joint_value
 
 
+def step_size_post_processor(values: List[NumberLike], _) -> pd.Series:
+    """Computes the largest feasible step size for each simulant. This is the smallest component-modified
+    step size, or the global step size, whichever is larger. If no components modify the step size, we default
+    to the global step size.
+
+    Parameters
+    ----------
+    values
+        A list of step sizes
+
+    Returns
+    -------
+    pandas.Series
+        The largest feasible step size for each simulant
+
+
+    """
+    if len(values) == 1:
+        return values[0]
+    min_modified = pd.DataFrame(values[1:]).min(axis=0).astype("timedelta64[ns]")
```

Comment:
Can you explain the indexing here? What is special about the 0th item?

Reply:
values[0] in a `list_combiner` is always the source of the pipeline. We're taking out the source, which defined by the global step size, and all the component modifications to it. I'll explain why below